### PR TITLE
add gomock and mockgen to jenkinsfile

### PR DESCRIPTION
**Pull request by jzila**

### CODE REVIEW

File: libkbfs/node_cache_test.go [hunk 1]
```diff
@@ -464,11 +464,9 @@ func TestNodeCacheGCReal(t *testing.T) {
 
 	childNode1 = nil
 	runtime.GC()
-	_ = <-finalizerChan
+	<-finalizerChan
 
-	if len(ncs.nodes) != 2 {
-		t.Errorf("Expected %d nodes, got %d", 2, len(ncs.nodes))
-	}
+	require.Len(t, ncs.nodes, 1)
```
Comment (jzila) [inline libkbfs/node_cache_test.go]: Fixed the test; @strib want to validate that this is okay? Seems to be consistent new GC behavior.

Comment (strib) [reply libkbfs/node_cache_test.go]: Seems weird to me -- which extra node is it GCing? I guess it must be `childNode2`, but it's supposed to keep that live until the end of the function.  Maybe I'll try `runtime.Keepalive` and see if it helps.

Comment (jzila) [reply libkbfs/node_cache_test.go]: Aren't there 3 nodes to start? There's `childNode1`, `childNode2`, and ...?

Comment (strib) [reply libkbfs/node_cache_test.go]: And the parent node, which both other nodes have references to.

**Pull request merged**